core/src/init/button/size.module.css

```diff
@@ -0,0 +1,33 @@
+.large {
+	height: 40px;
+	padding: 9px 16px;
```

Comment:
The height of a button should be defined via either "height" or vertical "padding", should not both. In our case, we prefer the height to be defined explicitly, and leave the vertical padding at zero
```suggestion
	height: 40px;
	padding: 0px 16px;
```